### DISCUSSION TITLE
v3: Falsche Tab-ID

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1075] [Show and tell]
Normalerweise ist die TAB-ID = Tabtitel + Nummer
![image](https://user-images.githubusercontent.com/84818934/135743978-4a406740-acf1-493a-8ff8-7592be054db8.png)
![image](https://user-images.githubusercontent.com/84818934/135743997-44037c3f-815f-47e8-b814-15287681a6ae.png)
![image](https://user-images.githubusercontent.com/84818934/135744047-06b79afd-b433-4168-91b6-4ca947bc0605.png)

Jetzt steht plötzlich eine andere längere ID drin, bei neu angelegten TABs seit `v3.0.0-alpha.84` ?
![image](https://user-images.githubusercontent.com/84818934/135744011-84f13caa-06b6-4e38-b2f2-3c909d901da1.png)
Die Browser URL ist auch entprechend anders:
![image](https://user-images.githubusercontent.com/84818934/135744034-d57da771-213a-4177-806d-d9c569e33d7b.png)

**Comment by Zefau** [comment]
Korrekt. Ist eine eindeutige sogenannte UUID, um sicherzugehen, dass die ID eindeutig ist. Die ID kann via JSON geändert werden.

**Comment by mcuiobroker** [comment]
Das ist aber auch ID, die man bei "setTabId" nutzt, um zwischen den TABs zu wechseln. Wie kann ich die id ändern? Wo?
Einfach dort eine andere ID reinschreiben?
![image](https://user-images.githubusercontent.com/84818934/135753182-511631d7-dae4-49c6-91d4-2583b9a23bd5.png)

**Comment by Zefau** [comment]
Im Layout JSON

**Comment by Zefau** [comment]
Jo, sieht richtig aus.

**Comment by mcuiobroker** [comment]
Warum wird denn plötzlich so eine lange ID vergeben, ändert sich das wieder?

**Comment by Zefau** [comment]
Das bleibt so. Muss eindeutig sein, sonst kommt es zu Problemen in jarvis

**Comment by mcuiobroker** [comment]
Die war doch vorher auch eindeutig. Mit TAB-Titel + Nummer. Wie soll man denn jetzt zwischen den TABs wechseln mit der langen ID?
Ich hab die jetzt bei mir auf kalender13 geändert (so wäre es normal gewesen) und hab keine Probleme.
